### PR TITLE
Got some initial READMEs, definitely needs more work though.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# 🚀 Short Term ETL Pipeline
```
Comment (Leon-JS02) [inline dashboard/README.md:1]: Nice rocket

**Review by Leon-JS02 — APPROVED** [review]
These are definitely good changes to the read me. very informative.

File: long_term/load_long.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-"""Given a file name uploads local json file to S3 bucket."""
+"""Given a file name uploads local csv file to S3 bucket."""
```
Comment (YakMan101) [inline long_term/load_long.py:1]: Good catch on the correction.

File: short_term/README.md [hunk 3]
```diff
@@ -26,7 +26,5 @@ BUCKET_NAME=XXXXXX
 ```
 1. Run `bash create_db.sh` to initialise the database schema.
 2. Create a python environment and run `pip install -r requirements.txt`.
-2. Run `python3 seed.py` to seed the database with initial metadata about plants, botanists, and locations.
-3. Build the Docker image, and upload to AWS ECR as a lambda.
-
-
+3. Run `python3 seed.py` to seed the database with initial metadata about plants, botanists, and locations.
+4. Run `bash dockerise.sh` to Dockerise and upload the container to the ECR repo.
```
Comment (YakMan101) [inline short_term/README.md:30]: Nice simple and direct.

**Review by YakMan101 — APPROVED** [review]

**Pull request merged**